pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -201,6 +201,11 @@ func (r *Reconciler) runReconcileSeedFlow(
 		return err
 	}
 
+	controllerRegistrationList := &gardencorev1beta1.ControllerRegistrationList{}
+	if err := r.GardenClient.List(ctx, controllerRegistrationList); err != nil {
+		return err
```

Comment:
fmt.Errorf

Reply:
Not sure what `fmt.Errorf` should tell at this point, other than it failed to list `ControllerRegistration`s which should already be part of the error message returned by the API server 🤔 

We follow the pattern of directly returning the error of trivial API calls in many places (see other List and Get calls in this file) if we can't add more meaningful context. WDYT?